blocks/search/search.css

```diff
@@ -1,163 +1,167 @@
-.search__box {
-  display: flex;
-  position: relative;
-  justify-content: flex-end;
-}
-
-.search__input-wrapper {
-  transition: all 130ms ease;
-  display: flex;
-  flex-direction: row;
-  width: 0;
-  opacity: 0;
-  right: 2rem;
-}
+.search {
```

Comment:
this whole file is green because I added the wrapper class to scope the variables; it's not all changed, but let me call out the changes for you explicitly